app/javascript/src/components/Common/Menubar.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from "react";
+
+import { Search, Settings, Plus } from "neetoicons";
+import { Typography } from "neetoui";
+import { MenuBar } from "neetoui/layouts";
```

Comment:
Why you have you imported from `neetoui` at some places and `@bigbinary/neetoui` at some other places? Stick onto one depending on the webpack aliases you've set.